flink-formats/flink-orc/pom.xml

```diff
@@ -182,7 +182,7 @@ under the License.
 			<type>test-jar</type>
 		</dependency>
 
-		<!-- ArchUit test dependencies -->
+		<!-- ArchUnit test dependencies -->
```

Comment:
@JingGe Thanks for the fix. But there are many similar typos and you can find them by global search. (e.g., https://github.com/apache/flink/blob/667d3a4cf7f59eb488dbbc4d4333df5220c85158/flink-formats/flink-compress/pom.xml#L94, https://github.com/apache/flink/blob/667d3a4cf7f59eb488dbbc4d4333df5220c85158/flink-table/flink-table-planner/pom.xml#L153, etc.)
We'd better fix them together.

Reply:
Thanks @TanYuxin-tyx for the review. I have addressed it. Please check if there is still any one missed. Thanks!